packages/EasyWebhook/src/Exceptions/CannotRerunWebhookException.php

```diff
@@ -4,7 +4,10 @@
 
 namespace EonX\EasyWebhook\Exceptions;
 
-final class CannotRerunWebhookException extends AbstractDoNotHandleMeException
+use Symfony\Component\Messenger\Exception\UnrecoverableExceptionInterface;
+
+final class CannotRerunWebhookException extends AbstractDoNotHandleMeException implements
```

Comment:
@alexndlm Thank you for this PR, I completely understand why, but I don't think that this is the right way to implement it as this solution couples the package exception with a Symfony one outside of the bridge namespace. We must find another way from within the bridge code to achieve it.

Reply:
@natepage have done.